README.md

```
@@ -1,2 +1,84 @@
+
+開発環境をセットアップする前に、以下の環境変数を設定する必要があります。  
+`.env`ファイルをプロジェクトのルートディレクトリに作成し、以下の内容を参考に設定ください。
+
+| 環境変数名          | 説明                   | ローカルでの推奨値        |
+|-------------------|----------------------|-------------------------|
+| `POSTGRES_NAME`   | Postgresのデータベース名 | `postgres`              |
+| `POSTGRES_USER`   | Postgresのユーザー名    | `postgres`              |
+| `POSTGRES_PASSWORD` | Postgresのパスワード    | `postgres`              |
+| `POSTGRES_DB`   | PostgresのDB名     | `db`                    |
+| `DEBUG`   | DEBUGモードで起動するかどうか（TRUEはDEBUGモード）.デフォルトはFalse    | `true`                  |
+|
```

Markdownlint (0.37.0)

Expected: 3; Actual: 1; Too few cells, row will be missing data
Table column count

(MD056, table-column-count)